### ISSUE TITLE
`cifar_torch.py` example unstable for `mode!=1, classifer=linear`

### ISSUE THREAD

**Issue by MuawizChaudhary** [kymatio/kymatio#1023]
The `cifar_torch.py` example for 2d ([Located here](https://github.com/kymatio/kymatio/blob/main/examples/2d/cifar_torch.py)) gives me nan's 4 epochs into the training when arguments `mode!=1` and `classifier=linear`. however, comments in the file indicate that we should obtain 70+ accuracy. Was a different set of hyper-parameters supposed to be used here?

How do we proceed here?

**Comment by janden** [comment]
That code looks like it needs some love in general. Do we know which paper it was based on?

**Comment by MuawizChaudhary** [comment]
I believe it was based on [Scattering Networks for Hybrid Representation Learning](https://arxiv.org/abs/1809.06367). @edouardoyallon could prolly specify.
